src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/Blocks/product_preferences_general.html.twig

```diff
@@ -35,12 +35,19 @@
     <div class="card-block row">
       <div class="card-text">
         <div class="form-group row">
-          {{ ps.label_with_help(('Catalog mode'|trans), ('The catalog mode is actually to disable products checkout (prices, add to cart, etc.) on your shop, like a retail website does.'|trans({}, 'Admin.Shopparameters.Help'))) }}
+          {{ ps.label_with_help(('Catalog mode'|trans), ('Catalog mode makes your store look like a retail website.'|trans({}, 'Admin.Shopparameters.Help'))) }}
```

Comment:
> Catalog mode makes your store look like a retail website.

Not sure about this wording. Isn't it supposed to disable checkout? How about this:

> Catalog mode disables the shopping cart on your store. Visitors will still able to browse your product catalog, but not buy them.